WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -511,12 +511,19 @@ final class EditableOrderViewModel: ObservableObject {
                                                                addressData: .init(billingAddress: orderSynchronizer.order.billingAddress,
                                                                                   shippingAddress: orderSynchronizer.order.shippingAddress),
                                                                onAddressUpdate: { [weak self] updatedAddressData in
-            let input = Self.createAddressesInput(from: updatedAddressData)
+            let input = Self.createAddressesInputIfPossible(billingAddress: updatedAddressData.billingAddress,
```

Comment:
TIL to call a static method by using `Self.` to reference the type itself, rather than the `self.` instance :D